.gx/lastpubver

```diff
@@ -1 +1 @@
-0.0.0: QmQTTqQnyvWoxM7gXWL1ptQ89nttDb6WsmTttnqz7G3tNf
+0.0.0: QmVF5rqMLt7sU1gfgCvwpNR4naDJ9TuvKN39WGxd9GYtx5
```

Comment:
We need to bump the version before releasing. You can use `gx release patch` to do that.